collect_app/src/main/res/layouts/fieldsight/layout/layout_simple_recycler_with_nodata.xml

```diff
@@ -71,7 +71,7 @@
             android:layout_height="wrap_content"
             android:layout_alignParentBottom="true"
             android:background="@color/primaryColor"
-            android:drawableRight="@drawable/ic_keyboard_arrow_right_24dp"
+            android:drawableRight="@drawable/ic_action_right"
```

Comment:
can we rename it to `ic_keyboard_arrow_right_legacy_24dp`?
That way we will know if we are importing svg/png? in the future